frontend/src/main/scala/bloop/cli/CliOptions.scala

```diff
@@ -14,6 +14,8 @@ case class CliOptions(
     version: Boolean = false,
     @HelpMessage("If set, print out debugging information to stderr.")
     verbose: Boolean = false,
+    @HelpMessage("If set, do not color output.")
+    noColor: Boolean = false,
```

Comment:
Would it be better to respect the environment variable `NO_COLOR`? http://no-color.org/

Reply:
Indeed, this would be a great default. I'll commit a message adding it so that we can unblock this pull request next week.